device/sycl/src/seeding/triplet_counting.sycl

```diff
@@ -135,7 +135,8 @@ class TripletCount {
         }
 
         if (mt_end_idx >= mid_top_doublets_per_bin.size()) {
-            mt_end_idx = fmin(mid_top_doublets_per_bin.size(), mt_end_idx);
+            mt_end_idx =
+                ::sycl::min(mid_top_doublets_per_bin.size(), mt_end_idx);
```

Comment:
Ahh, I actually forgot about these... @beomki-yeo, I fear that @konradkusiak97 copied these `fmin(...)` calls from the CUDA code directly.

But the code compares two integers here. :confused: So `fmin(...)` is not appropriate. Even more weird, why do this behind an `if(...)` statement? Once we're in there, we **know** that the value has to be replaced, no? :confused:

Reply:
Indeed. That's probably one of dumb codes that I've written